mesa/datacollection.py

```diff
@@ -34,11 +34,13 @@
     * The schedule has an agent list called agents
     * For collecting agent-level variables, agents must have a unique_id
 """
+import contextlib
 import itertools
 import types
 from functools import partial
 
-import pandas as pd
+with contextlib.suppress(ImportError):
```

Comment:
Reading https://stackoverflow.com/questions/34566806/why-use-contextlib-suppress-as-opposed-to-try-except-with-pass, I suppose that using `contextlib.suppress` is more concise than try-except.